src/aws.yml

```diff
@@ -1,11 +1,22 @@
 ---
-- hosts: aws
+- hosts: all
   name: AWS-specific roles
   become: yes
   become_method: sudo
   roles:
-    - amazon_efs_utils
     - cloudwatch_agent
     # The instance types used for almost all the instances expose EBS
     # volumes as NVMe block devices, so that's why we need nvme here.
     - nvme
+    # This role forces the installation of python2, which breaks any
+    # pip modules that are installed afterwards because Ansible will
+    # see that /usr/bin/python exists and it.  In particular, Ansible
```

Comment:
I believe this should be "exists and use it.".

Reply:
See commit 32756d4.